serverless/event_workflows/serverless-sinkbinding.adoc

```diff
@@ -10,10 +10,30 @@ SinkBinding is used to connect event producers, or _event sources_, to an event
 
 [NOTE]
 ====
-Both of the following procedures require you to create YAML files.
+The following procedures require you to create YAML files.
 
 If you change the names of the YAML files from those used in the examples, you must ensure that you also update the corresponding CLI commands.
 ====
 
+To use a SinkBinding, cluster administrators must label the namespace configured in the SinkBinding with `bindings.knative.dev/include:"true"`:
+
+[source,terminal]
+----
+$ oc label namespace <namespace> bindings.knative.dev/include=true
+----
+
+Users must manually add a `bindings.knative.dev/include=true` label to resources.
```

Comment:
If you have it, I think that a tiny bit of context should go after "resources."

Reply:
I think now that this is in the modules it's clear enough, also in the context of Serverless docs I think we can assume a level of knowledge here.

Reply:
You're right - I read the rest of the module, and the user should understand why they're doing this, especially if you add the requirement to add the label to the existing YAML.